crates/wash-cli/src/drain.rs

```diff
@@ -1,11 +1,23 @@
 use std::collections::HashMap;
+use std::path::Path;
 
 use anyhow::Result;
 use serde_json::json;
 use wash_lib::cli::CommandOutput;
 use wash_lib::drain::Drain;
 
-pub fn handle_command(cmd: Drain) -> Result<CommandOutput> {
+pub fn handle_command(cmd: Drain) -> Result<CommandOutput, anyhow::Error> {
+    if matches!(cmd, Drain::All | Drain::Downloads) {
+        let wasmcloud_pid_path =
+            Path::new(&dirs::home_dir().unwrap()).join(".wash/downloads/wasmcloud.pid");
+        let wadm_pid_path = Path::new(&dirs::home_dir().unwrap()).join(".wash/downloads/wadm.pid");
+
+        if wasmcloud_pid_path.exists() || wadm_pid_path.exists() {
+            return Err(anyhow::Error::msg(
+                "Cannot clear out the caches because wasmcloud is still up and running",
```

Comment:
here we can use `anyhow::bail` probably, I think. 

And, you might want to include `wadm` in the message since we chance for either `wasmcloud` or `wadm` instances.

We also might want to be *really* clear that the PID files are still on disk -- sometimes those files get left around and someone might not know *why* we think it's still running

Reply:
I understand :+1: , I'll convert Error -> bail and Change the message to focus on the existence of PID files

Reply:
`Cannot clear the caches : wasmcloud and wadm PID files are still present on disk, processes might still be active or were not properly terminated.`

Is this message alright ?